File: too_many_cooks/lib/too_many_cooks.dart
/// Multi-agent Git coordination MCP server.
///
/// Enables multiple AI agents to safely edit a git repository simultaneously
/// through advisory file locking, identity verification, inter-agent messaging,
/// and plan visibility.
library;

export 'package:too_many_cooks_data/too_many_cooks_data.dart'
    show TooManyCooksDb, createDb;

export 'src/admin_routes.dart';
export 'src/config.dart';
export 'src/server.dart';
export 'src/types.dart';

File: too-many-cooks/lib/too_many_cooks.ts
/// Multi-agent Git coordination MCP server.
///
/// Enables multiple AI agents to safely edit a git repository simultaneously
/// through advisory file locking, identity verification, inter-agent messaging,
/// and plan visibility.

export * from "./src/admin_routes.js";
export * from "./src/config.js";
export { type TooManyCooksDb, createDb } from "./src/data/data.js";
export { type AgentEventHub, createAgentEventHub } from "./src/notifications.js";
export * from "./src/server.js";
export * from "./src/types.js";
